Models/Team.cs

```diff
@@ -7,10 +7,10 @@ public class Team
         public int Id { get; set; }
         public string Name { get; set; }
         public string Division { get; set; }
-        public List<Player> Players { get; set; }
+        public List<int> Players { get; set; }
         public List<int> GameScores { get; set; }
-        public List<Game> PlayInGames { get; set; }
-        public List<Game> BracketGames { get; set; }
+        public List<int> PlayInGames { get; set; }
+        public List<int> BracketGames { get; set; }
```

Comment:
We can leave these as `List<Player>` and `List<Game>`. `TeamService` should handle mapping from an id stored in MySQL to an actual `Player` or `Game` object.

Reply:
fixed